k8s/charts/openebs/templates/validationwebhook.yaml

```diff
@@ -42,13 +42,6 @@ metadata:
     chart: {{ template "openebs.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-  # Helm hook annotations in order to ensure that the certs
```

Comment:
Why is this removed?

Reply:
Thus pre-install hook prevents the secret creation if someone uses the older version of charts and upgraded to new version.
[https://github.com/helm/charts/pull/14162](url)